src/doc/DocAnnotator.js

```diff
@@ -488,7 +488,7 @@ class DocAnnotator extends Annotator {
      */
     resetHighlightSelection(event) {
         const isCreateDialogVisible = this.createHighlightDialog && this.createHighlightDialog.isVisible;
-        if (!isCreateDialogVisible || util.isInDialog(event)) {
+        if (!isCreateDialogVisible || !event || util.isInDialog(event)) {
```

Comment:
Question about this: Does this mean you can no longer remove highlight selection in a function that isn't triggered by a browser event?

Reply:
No. There are still one or 2 occasions where the highlight selection is removed without destroying the dialog. This is more for automatically destroying the dialog AND clearing the selection